synapse/tests/test_model_files.py

```diff
@@ -22,6 +23,68 @@ def test_model_filebytes(self):
             self.raises(s_exc.BadTypeValu, fbyts.norm, 'helo:moto')
             self.raises(s_exc.BadTypeValu, fbyts.norm, f'sha256:{s_common.guid()}')
 
+    def test_model_filebytes_pe(self):
```

Comment:
From a test overhead perspective can we roll these tests up into one?

Reply:
yes